application/src/main/java/org/togetherjava/tjbot/commands/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -37,39 +35,29 @@ public void onMessageReceived(MessageReceivedEvent event) {
             return;
         }
 
-        if (messageHasNoMediaAttached(event)) {
-            deleteMessage(event).flatMap(any -> dmUser(event)).queue();
-        }
-    }
-
-    private static boolean messageHasNoMediaAttached(MessageReceivedEvent event) {
         Message message = event.getMessage();
-        return message.getAttachments().isEmpty() && message.getEmbeds().isEmpty()
-                && !message.getContentRaw().contains("http");
-    }
-
-    private AuditableRestAction<Void> deleteMessage(MessageReceivedEvent event) {
-        return event.getMessage().delete();
-    }
-
-    private RestAction<Message> dmUser(MessageReceivedEvent event) {
-        return dmUser(event.getMessage());
-    }
-
-    private RestAction<Message> dmUser(Message originalMessage) {
-        String originalMessageContent = originalMessage.getContentRaw();
-        MessageEmbed originalMessageEmbed =
-                new EmbedBuilder().setDescription(originalMessageContent)
-                    .setColor(Color.ORANGE)
-                    .build();
-
-        Message dmMessage = new MessageBuilder(
-                "Hey there, you posted a message without media (image, video, link) in a media-only channel. Please see the description of the channel for details and then repost with media attached, thanks 😀")
-                    .setEmbeds(originalMessageEmbed)
-                    .build();
+        if (message.toString().contains("THREAD_CREATED")) {
+            return;
+        }
 
-        return originalMessage.getAuthor()
-            .openPrivateChannel()
-            .flatMap(channel -> channel.sendMessage(dmMessage));
+        if (message.getAttachments().isEmpty() && message.getEmbeds().isEmpty()
```

Comment:
rly dislike the method deletion and the whole logic into one big clunky section, i prefer that you extract methods